src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioDiagnosticListTable.BuildTableDataSource.cs

```diff
@@ -193,6 +193,9 @@ public override bool TryGetValue(int index, string columnName, out object conten
                         case StandardTableKeyNames.ErrorCodeToolTip:
                             content = GetHelpLinkToolTipText(data);
                             return content != null;
+                        case StandardTableKeyNames.HelpKeyword:
```

Comment:
There is a lot of redundancy in this method.  Why not have a "GetValue" method that just returns things like ```data.id```.  Then have htis method be:

```C#
bool TryGetValue(...)
{
    var value = GetValue(...);
    return value != null;
}
```

It should cut down the size of this method (and the repeated code) by nearly half.